File: 11720.cpp
#include <stdio.h>
#include <stdlib.h>

int main() {
    int cnt, sum = 0; 
    scanf("%d", &cnt);
    char* num = (char*)calloc(cnt, sizeof(char));
    
    scanf("%s", num); 
    for(int i = 0; i < cnt; i++)
        sum += (num[i] - '0');
    
    printf("%d\n", sum); 
    free(num); 
    return 0;
}

File: 11720.java
import java.util.Scanner;
public class Main {
    public static void main(String args[]) {
        int cnt, sum = 0;
        String num; 
        Scanner sc = new Scanner(System.in);
        cnt = sc.nextInt();
        num = sc.next(); 
        
        for(int i = 0; i < cnt; i++) {
            sum += Integer.parseInt(num.substring(i, i + 1));
        }
        System.out.println(sum); 
    }
}
